dev/benchmarks/macrobenchmarks/android/app/src/main/AndroidManifest.xml

```diff
@@ -22,6 +22,7 @@ found in the LICENSE file. -->
         android:icon="@mipmap/ic_launcher">
         <activity
             android:name=".MainActivity"
+            android:exported="true"
```

Comment:
I don't see any reason for this flag to be set. These benchmarks aren't exporting any services. Was there a reason this got set here?